src/smallestPrimeFactor.js

```diff
@@ -12,11 +12,27 @@
  * @returns {number} The smallest prime factor of num
  */
 function smallestPrimeFactor(num) {
-  // This is your job. :)
+  let changingNumber = Math.round(num / 2)
+  let smallestPrime = 0
+  while (changingNumber > 1) {
```

Comment:
If you start at `2` and go up, the first prime factor you find will be the smallest. You only need to iterate up to the smallest possible proper prime factor (which is smaller than `num / 2`).